app/src/main/java/com/framgia/toeic/screen/exam/ExamLessonAdapter.java

```diff
@@ -54,6 +57,7 @@ public Viewholder(View itemView, OnItemClickListener listener) {
             mImageView = itemView.findViewById(R.id.image_grammar);
             mTextTitle = itemView.findViewById(R.id.text_name_grammar);
             mCardView = itemView.findViewById(R.id.cardview_grammar);
+            mRatingBar = itemView.findViewById(R.id.ratingBar);
```

Comment:
```suggestion
            mRatingBar = itemView.findViewById(R.id.rating_bar);
```
thống nhất format name